cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -61,17 +61,17 @@
         }
     }
 
-    .list {
+    .m-list {
```

Comment:
Could we use [`m-list__horizontal'](https://cfpb.github.io/capital-framework/components/cf-typography/#horizontal-list-modifier) instead? Seems like this is duplicating those styles.